lightning/src/ln/interactivetxs.rs

```diff
@@ -1662,14 +1662,55 @@ impl InteractiveTxConstructor {
 	}
 }
 
+/// Determine whether a change output should be added or not, and if so, of what size,
+/// considering our given inputs, outputs, and intended contribution.
+/// Computes and takes into account fees.
```

Comment:
Could we mention in these docs the optional returned value is "remaining value" and describe what that means since we just return a `u64` here?